<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>747</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: white;
    font-family: &quot;Microsoft YaHei&quot;, Arial;
}
QLabel {
    color: #333333;
    font-size: 15px;
    font-weight: bold;
    padding: 5px 0;
    min-width: 120px;
    background: transparent;
}
QLineEdit {
    padding: 12px;
    border: 2px solid #dddddd;
    border-radius: 4px;
    background-color: white;
    font-size: 14px;
    min-height: 25px;
    color: #333333;
}
QLineEdit:hover {
    border: 2px solid #bbbbbb;
}
QLineEdit:focus {
    border: 2px solid #2196F3;
}
QLineEdit:disabled {
    background-color: #f5f5f5;
    color: #666666;
    border: 2px solid #eeeeee;
}
QPushButton {
    padding: 12px 30px;
    font-size: 14px;
    border: none;
    border-radius: 4px;
    color: white;
    min-width: 120px;
    min-height: 40px;
}
QPushButton#pushButton {
    background-color: #757575;
}
QPushButton#pushButton:hover {
    background-color: #616161;
}
QPushButton#pushButton_2 {
    background-color: #4CAF50;
}
QPushButton#pushButton_2:hover {
    background-color: #45a049;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="header_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#header_widget {
    background-color: #f5f5f5;
    border-bottom: 1px solid #dddddd;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="title_label">
        <property name="styleSheet">
         <string notr="true">font-size: 24px;
color: #333333;
padding: 20px 0;</string>
        </property>
        <property name="text">
         <string>用户信息</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="content_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>60</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>53</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>请输入用户名</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>53</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>请输入密码</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>真实姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>53</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>请输入真实姓名</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>权限</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>53</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    padding: 12px;
    border: 2px solid #dddddd;
    border-radius: 4px;
    background-color: white;
    font-size: 14px;
}
QComboBox:hover {
    border: 2px solid #bbbbbb;
}
QComboBox:focus {
    border: 2px solid #2196F3;
}
QComboBox:disabled {
    background-color: #f5f5f5;
    color: #666666;
    border: 2px solid #eeeeee;
}
QComboBox::drop-down {
    border: none;
    width: 30px;
}
QComboBox::down-arrow {
    image: url(resources/icons/dropdown.png);
    width: 12px;
    height: 12px;
}
QComboBox QAbstractItemView {
    border: 1px solid #dddddd;
    border-radius: 4px;
    background-color: white;
    selection-background-color: #f5f5f5;
}</string>
          </property>
          <item>
           <property name="text">
            <string>请选择权限</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学生</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>教师</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>管理员</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>学号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>53</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>请输入学号</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>64</height>
           </size>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>64</height>
           </size>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
